src/Plugin/EntityReferenceSelection/OgSelection.php

```diff
@@ -78,7 +56,7 @@ protected function buildEntityQuery($match = NULL, $match_operator = 'CONTAINS')
 
     $ids = [];
 
-    if ($this->configuration['handler_settings']['field_mode'] == 'admin') {
+    if ($this->isGroupAdmin()) {
```

Comment:
@RoySegall @amitaibu Just removed this as in D7 I think this was just done at runtime anyway? So no need for a config option here?